### DISCUSSION TITLE
How to realize the interval startup between two jobs

### DISCUSSION THREAD

**Discussion by piqihaha** [swar/Swar-Chia-Plot-Manager#118] [Q&A]
Hello, I'm from China. My English is not good. I can't even read the manual of configuration file. I use software to translate
My computer CPU: AMD 5800 8c16t, memory: 48g, SSD: P3600 1.2t * 2, HDD: 8t * 2
Some of my important profiles:

global:
max_ concurrent: 8
jobs:
  - name: A1
    max_plots: 999
    farmer_public_key: 
    pool_public_key: 
    temporary_directory: E:\    #（this is SSD 1.4T）
    temporary2_directory:
    destination_directory: H:\   #（this is HDD 8T）
    size: 32
    bitfield: true
    threads: 2
    buckets: 128
    memory_buffer: 4096
    max_concurrent: 4
    max_concurrent_with_start_early: 7
    stagger_minutes: 10
    max_for_phase_1: 2
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false

  - name: A2
    max_plots: 999
    farmer_public_key: 
    pool_public_key: 
    temporary_directory: F:\    #（this is SSD 1.4T）
    temporary2_directory:
    destination_directory: G:\    #（this is HDD 8T）
    size: 32
    bitfield: true
    threads: 2
    buckets: 128
    memory_buffer: 4096
    max_concurrent: 4
    max_concurrent_with_start_early: 3
    stagger_minutes: 15
    max_for_phase_1: 2
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false

The results I want are:
Job A1 and A2 start plot in turn with an interval of 10 minutes
Finally, A1 carries out 4 plots, A2 carries out 4 plots, a total of 8 plots

------------------------------------------------------

Or how can I configure it to give full play to the performance of my computer?
It can realize 10 or 8 plots at the same time?

**Comment by Amneamnius** [comment]
Try this, 4 Plots x 4 = 16 Threads each at a time in Phase1. If Phase3 times are bad lower threads from 4 --> 3 -

global:
max_ concurrent: 10

name: A1
threads: 4
max_concurrent: 4
max_concurrent_with_start_early: 5
stagger_minutes: 0
max_for_phase_1: 2

name: A2
threads: 4
max_concurrent: 4
max_concurrent_with_start_early: 5
stagger_minutes: 0
max_for_phase_1: 2

**Comment by piqihaha** [comment]
> Try this, 4 Plots x 4 = 16 Threads each at a time in Phase1. If Phase3 times are bad lower threads from 4 --> 3 -
> 
> global:
> max_ concurrent: 10
> 
> name: A1
> threads: 4
> max_concurrent: 4
> max_concurrent_with_start_early: 5
> stagger_minutes: 0
> max_for_phase_1: 2
> 
> name: A2
> threads: 4
> max_concurrent: 4
> max_concurrent_with_start_early: 5
> stagger_minutes: 0
> max_for_phase_1: 2

-----------------------------

threads: 2
buckets: 128
memory_buffer: 4096

What about threads and memory? Keep it that way?

**Comment by piqihaha** [comment]
> Try this, 4 Plots x 4 = 16 Threads each at a time in Phase1. If Phase3 times are bad lower threads from 4 --> 3 -
> 
> global:
> max_ concurrent: 10
> 
> name: A1
> threads: 4
> max_concurrent: 4
> max_concurrent_with_start_early: 5
> stagger_minutes: 0
> max_for_phase_1: 2
> 
> name: A2
> threads: 4
> max_concurrent: 4
> max_concurrent_with_start_early: 5
> stagger_minutes: 0
> max_for_phase_1: 2

I've just started testing with this profile, try it later with your profile
  global:
  max_concurrent: 8

  - name: avalon
    max_plots: 8
    farmer_public_key: 
    pool_public_key: 
    temporary_directory: E:\
    temporary2_directory: F:\
    destination_directory: 
      - H:\
      - G:\
    size: 32
    bitfield: true
    threads: 2
    buckets: 128
    memory_buffer: 4096
    max_concurrent: 8
    max_concurrent_with_start_early: 8
    stagger_minutes: 10
    max_for_phase_1: 8
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false

**Comment by Amneamnius** [comment]
Use a program like hwinfo to track % Free Memory and increase memory_buffer if there is a lot of space left

**Comment by piqihaha** [comment]
> Use a program like hwinfo to track % Free Memory and increase memory_buffer if there is a lot of space left
![666](https://user-images.githubusercontent.com/25221169/117118837-d9d86e80-adc3-11eb-88a4-91e0ef2dc602.png)

It doesn't look right, I'm 8 plots now, but the writes are all in temp1 SSD, temp2 SSD is empty, there's no concurring, so I feel like it's going to affect performance, and SSD1 is 1.4T, it's going to be full right now, so it's going to crash?

The current configuration is:
global:
max_concurrent: 8

name: avalon
max_plots: 8
farmer_public_key:
pool_public_key:
temporary_directory: E:
temporary2_directory: F:
destination_directory:
H:\
G:
size: 32
bitfield: true
threads: 2
buckets: 128
memory_buffer: 4096
max_concurrent: 8
max_concurrent_with_start_early: 8
stagger_minutes: 10
max_for_phase_1: 8
concurrency_start_early_phase: 4
concurrency_start_early_phase_delay: 0
temporary2_destination_sync: false


So I feel like I'm going to use two jobs, one SSD and one HDD for each. But I don't know how to modify the profile

**Comment by Amneamnius** [comment]
Yes, do it the way you had it on first post and if you want increase threads like I had.

**Comment by swar** [comment]
Please use Discussions if this is not pertaining to a software issue.
